src/content/docs/blog/tauri-2-0-0-beta.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Announcing the Tauri v2 Beta Release
+date: 2024-02-02
+authors: [lucasfernog]
+excerpt: Tauri v2 is feature complete! 2.0.0-beta.0 has been published.
+banner:
+  content: |
+    You're reading the blog on the prerelease site for Tauri 2.0 -
+    <a href="https://tauri.app">Go to the Tauri 1.0 site</a>
+---
+
+![Hero Image](./tauri_2_0_0_alpha_0/header.png)
+
+Tauri v2 has been in progress for [over a year](/blog/tauri-mobile-alpha) and it is now feature complete! We have just released [Tauri v2.0.0-beta.0](https://github.com/tauri-apps/tauri/releases/tag/tauri-v2.0.0-beta.0) which represents a major milestone from our [roadmap](/blog/roadmap-to-tauri-2-0#beta).
```

Comment:
i wonder if we can phrase that "feature complete" part better somehow 🤔 i'm scared that people may interpret this as we consider _mobile_ support feature complete and then they will riot cause there are so many features missing x)

I'm fine with keeping it as is though (also can't think of another phrasing myself) just wanted to bring it up. Also, we're clearing up the missing mobile features further down so maybe i'm just bitchy again.

Reply:
I agree, "feature complete" sounds a bit much
- "and is now ready for public scrutiny"
- "and is now suitable for human consumption"
- "and is now ready to take the next step towards being stable"
- "and we're happy to announce that we're nearing the end of the wait"
- "and is now officially better than Electron"

Reply:
i wish we could use all of those suggestions at the same time, they are great 😂

Reply:
good idea, let's mention that we know we still need some mobile APIs